lib/chess/core/chess.rb

```diff
@@ -31,7 +31,11 @@ module Chess
   EXIT_COMMAD = :exit_game
   COMMAND_SUCCES = :ok
 
-  CASTLING_FAILS = :cant_castle
+  CASTLING_FAILS = :cant_castling
+
+  INTENTION_IS_MOVE = :move
+  INTENTION_IS_KING_CASTLING = :castling_king_side
+  INTENTION_IS_QUEEN_CASTLING = :castling_queen_side
```

Comment:
I suggest renaming these constants for clarity. Constants should be nouns, not verbs... or sentences
```suggestion
  MOVE_INTENTION = :move
  KING_CASTLING_INTENTION = :castling_king_side
  QUEEN_CASTLING_INTENTION = :castling_queen_side
```

Reply:
right, done!